src/content/docs/en/guides/integrations-guide/sitemap.mdx

```diff
@@ -475,6 +475,40 @@ export default defineConfig({
 
 The given configuration will generate sitemap files at `https://example.com/astronomy-sitemap-0.xml` and `https://example.com/astronomy-sitemap-index.xml`.
 
+### `namespaces`
+
+<p>
+
+**Type:** `{ news?: boolean; xhtml?: boolean; image?: boolean; video?: boolean; }` <br />
+**Default:** `{ news: true, xhtml: true, image: true, video: true }` <br />
+<Since v="3.6.0" pkg="@astrojs/sitemap" />
+</p>
+
+An object of XML namespaces to exclude from the generated sitemap.
+
+Excluding unused namespaces can help create cleaner, more focused sitemaps that are faster for search engines to parse and use less bandwidth. For example, if your site doesn't have news content, videos, or multiple languages, you can exclude those namespaces to reduce XML bloat.
+
+By default, all configurable namespaces  (`news`, `xhtml`, `image`, and `video`) are included in your generated sitemap XML. To exclude a namespace from your sitemap generation, add a `namespaces` configuration object and set individual options to `false`:
+
+```js title="astro.config.mjs" ins={8-14}
+import { defineConfig } from 'astro/config';
+import sitemap from '@astrojs/sitemap';
+
+export default defineConfig({
+  site: 'https://example.com',
+  integrations: [
+    sitemap({
+      // Only include image and video namespaces
+      namespaces: {
+        news: false,
+        xhtml: false, 
+        // image and video will be true by default
```

Comment:
```suggestion
      namespaces: {
        news: false,
        xhtml: false, 
```
I think this code snippet works even without the comments! I had originally thought about keeping one or the other, but I'm not sure it's needed.